docs/src/proving_systems/starks/api.md

```diff
@@ -72,14 +68,14 @@ The way we specify our fibonacci transition constraint looks like this:
 
 ```rust
 fn compute_transition(
-        &self,
-        frame: &air::frame::Frame<Self::Field>,
-    ) -> Vec<FieldElement<Self::Field>> {
+    &self,
+    frame: &air::frame::Frame<Self::Field>,
+) -> Vec<FieldElement<Self::Field>> {
     let first_row = frame.get_row(0);
     let second_row = frame.get_row(1);
     let third_row = frame.get_row(2);
 
-    vec![third_row[0] - second_row[0] - first_row[0]]
+    vec![third_row[0].clone() - second_row[0].clone() - first_row[0].clone()]
```

Comment:
This is the compute of quadratic constraints, right ? But the documentation. It can be changed for 

`vec![&second_row[0] - &first_row[0] * &first_row[0]]`

No need for clones

Reply:
I removed the `clone`s, Thanks!